src/components/Catalog/index.scss

```diff
@@ -108,4 +213,48 @@
     &__arrow{
         padding:0px 32px 32px 0px;
     }
-}
\ No newline at end of file
+}
+//SALE CARD
+.sale-card {   
+    cursor: pointer;
+
+    &__container{
+        display: flex;
+        width: 720px;
+        height: 190px;
+    }
```

Comment:
1. check everywhere on white space line between styles (like here &__container and &__description{